### PR TITLE
v13 data release

### PR DESCRIPTION
**Pull request by jharenza**
#### What scientific question is your analysis addressing?
V13 data release

#### What was your approach?
NA

#### What GitHub issue does your pull request address?
#373 

#### Which areas should receive a particularly close look?
This is not ready yet (files will be uploaded to S3 after 2PM), but wanted to push the documents through for a look to determine if anything has to change since now we are adding both `release-notes.md` and `data-file-descriptions.md` in two places, within the release and in the `doc/` folder. Does everything seem appropriately addressed?

#### Is there anything that you want to discuss further?
@cansavvy can you take a look at the `data-file-descriptions.md` for the intervals used for TMB?
Are analysis files adequately described?

@jaclyn-taroni I added STAR output files to the gene expression section, but MendQC to a new QC section - does this sound reasonable?

### Data Release Checklist
- [x] Is the table in doc/data-file-descriptions.md up to date?
- [x] Is doc/data-format.md up to date?
- [x] Is doc/release-notes.md up to date?
- [x] Is download-data.sh up to date?

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -1,6 +1,115 @@
...
+    - pbta-mend-qc-manifest.tsv
+    - pbta-star-log-final.tar.gz
+    - pbta-star-log-manifest.tsv
+  - Add TCGA MAF and clinical files per [#257](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/257). Files added:
+    - pbta-tcga-snv-lancet.vep.maf.gz
+    - pbta-tcga-snv-mutect2.vep.maf.gz
+    - pbta-tcga-snv-strelka2.vep.maf.gz
+    - pbta-tcga-manifest.tsv
+  - Add `cnv-consensus.tsv` file from [#128](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/128).
+  - Add analysis files from [](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/351#issuecomment-571295623)
```
Comment (cgreene) [inline doc/release-notes.md]: empty brackets here

Comment (cgreene) [inline doc/release-notes.md]:
```suggestion
  - Add analysis files from [#351](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/351#issuecomment-571295623)
```

**Comment by yuankunzhu** [comment]
@jharenza should the release in the data-files-description.md be v13? https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/444/files#diff-0468bf9add309a2a34494766546a847aL18

**Comment by jharenza** [comment]
> @jharenza should the release in the data-files-description.md be v13? https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/444/files#diff-0468bf9add309a2a34494766546a847aL18

yes, correct - updated

File: doc/data-formats.md [hunk 2]
```diff
@@ -136,6 +149,11 @@ Consensus mutation files are products of the [`analyses/snv-callers`](https://gi
   * `pbta-snv-consensus-mutation-tmb-all.tsv` includes tumor mutation burden statistics that are calculated based calculated from Strelka2 and Mutect2 SNV consensus, and the intersection of Strelka2 and Mutect2 BED windows sizes.
   * `pbta-snv-consensus-mutation-tmb-coding.tsv` contains coding only tumor mutation burden statistics calculated from the number of coding sequence Strelka2, Mutect2, and Lancet consensus SNVs and size of the intersection of all three callers' BED windows and the Gencode v27 coding sequence regions. 
 
+### Consensus Copy Number File
```
Comment (jaclyn-taroni) [inline doc/data-formats.md]: Can we group this with the GISTIC information below (line 190) please?

File: doc/data-formats.md [hunk 3]
```diff
@@ -7,17 +7,26 @@ A table with brief descriptions for each data file is provided in the `data-desc
 
 PBTA data files are all files derived from samples (e.g., tumors, cell lines) that are processed upstream of this repository and are not the product of any analysis code in the `AlexsLemonade/OpenPBTA-analysis` repository.
 
+### Quality Control Data
 
+MendQC output files `*readDist.txt` and `*bam_umend_qc.tsv`, along with a manifest mapping filename to biospecimen, are provided.
```
Comment (jaclyn-taroni) [inline doc/data-formats.md]: I would link to MendQC here.

File: doc/data-files-description.md [hunk 4]
```diff
@@ -27,7 +27,8 @@ This document contains information about all data files associated with this pro
...
-|`WXS.hg38.100bp_padded.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#snv-and-indel-calling) | hg38 WXS regions provided by the kit manufacturer used for all variant callers with each region padded by 100 bp
+|`WXS.hg38.100bp_padded.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#snv-and-indel-calling) | hg38 WXS regions provided by the kit manufacturer used for Strelka2, Mutect2, and VarDict variant callers with each region padded by 100 bp
+|`WXS.hg38.lancet.400bp_padded.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#snv-and-indel-calling) | hg38 WXS regions provided by the kit manufacturer used for Lancet variant callers with each region padded by 400 bp
```
Comment (jaclyn-taroni) [inline doc/data-files-description.md]: 300?

Comment (jharenza) [reply doc/data-files-description.md]: turns out, was 300 plus the 100 from the old BED :(

File: doc/data-formats.md (hunk 3 above)
Comment (jharenza) [reply doc/data-formats.md]: updated

File: doc/data-formats.md (hunk 2 above)
Comment (jharenza) [reply doc/data-formats.md]: updated

**Comment by jharenza** [comment]
@jaclyn-taroni the files are uploaded and md5sum updated. i am testing the download now, but should be ready for CI work.

**Comment by jharenza** [comment]
Data download successful 👍

**Pull request merged**